service/src/main/java/bio/terra/cbas/initialization/InstanceInitializerBeanConfig.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.cbas.initialization;
+
+import bio.terra.cbas.config.CbasContextConfiguration;
+import bio.terra.cbas.dao.MethodDao;
+import bio.terra.cbas.dao.MethodVersionDao;
+import bio.terra.cbas.dao.RunDao;
+import bio.terra.cbas.dao.RunSetDao;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
todo: is this needed?

Reply:
update: it turns out *is* needed

Reply:
I believe without this, Spring would never find the `@Bean` inside it